app/src/main/java/org/fossasia/openevent/app/core/session/create/CreateSessionPresenter.java

```diff
@@ -122,4 +125,12 @@ public void createSession(long trackId, long eventId) {
                 getView().dismiss();
             }, Logger::logError);
     }
+
+    public List<String> getSessionStateList() {
+        sessionStateList.add("draft");
+        sessionStateList.add("accepted");
+        sessionStateList.add("pending");
+        sessionStateList.add("confirmed");
+        return sessionStateList;
```

Comment:
You can add this directly in the resources since the choices are predetermined. More info here: https://developer.android.com/guide/topics/ui/controls/spinner